### PR TITLE
little refactor for consistency

### PR DESCRIPTION
**Pull request by pradeepitm12**
this adds minor refactor to maintain consistency with upcoming PRs

Signed-off-by: Pradeep Kumar <pradkuma@redhat.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Run the code checkers with `make check`
- [x] Regenerate the manpages, docs and go formatting with `make generated`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

File: pkg/actions/create.go [hunk 1]
```diff
@@ -21,8 +21,9 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
-func Create(gvr schema.GroupVersionResource, clients *cli.Clients, object *unstructured.Unstructured, ns string, op metav1.CreateOptions) (*unstructured.Unstructured, error) {
-	obj, err := clients.Dynamic.Resource(gvr).Namespace(ns).Create(object, op)
+func Create(gr schema.GroupVersionResource, clients *cli.Clients, object *unstructured.Unstructured, ns string, op metav1.CreateOptions) (*unstructured.Unstructured, error) {
+	gvr, _ := GetGroupVersionResource(gr, clients.Tekton.Discovery())
```
Comment (danielhelfand) [inline pkg/actions/create.go:25]: err catch here?

Comment (pradeepitm12) [reply pkg/actions/create.go:25]: Good Catch.

File: pkg/cmd/task/start.go [hunk 2]
```diff
@@ -89,7 +89,7 @@ func NameArg(args []string, p cli.Params) error {
 		return fmt.Errorf(errInvalidTask, name, ns)
 	}
 
-	if t.Spec.Resources != nil {
+	if t.Spec.Params != nil {
```
Comment (vdemeester) [inline pkg/cmd/task/start.go:92]: ohhh :sweat_smile: :cold_sweat:

**Pull request merged**